elementary/utils/models.py

```diff
@@ -0,0 +1,24 @@
+from elementary.monitor.alerts.model import ModelAlert
+from elementary.monitor.alerts.source_freshness import SourceFreshnessAlert
+from elementary.monitor.alerts.test import TestAlert
+
+
+def get_shortened_model_name(model):
+    if model is None:
+        # this can happen for example when a Singular test is failing for having no refs.
+        return None
+    return model.split(".")[-1]
+
+
+def alert_to_concise_name(alert):
```

Comment:
Think this should be on a utils folder under the alert section and not the global elementary project (because it is only relevant to alerts)

Reply:
Accept

Reply:
seems there's no local utils folder + most other utils in the global folder are also narrowly used

not accept

Reply:
we spoke about it and decided to leave it there